paddle/fluid/prim/utils/static/static_global_utils.h

```diff
@@ -69,12 +69,21 @@ class StaticCompositeContext {
     enable_bwd_prim_ = enable_prim;
   }
 
+  void SetTargetGradName(const std::map<std::string, std::string>& m) {
+    target_grad_name_ = m;
+  }
+
+  std::map<std::string, std::string> GetTargetGradName() {
```

Comment:
Maybe just give a check method instead of return whole map

Reply:
fixed in a new pr.